psgo.go

```diff
@@ -282,11 +283,13 @@ func JoinNamespaceAndProcessInfo(pid string, descriptors []string) ([][]string,
 		return nil, err
 	}
 
+	ctx := new(psContext)
+
 	// extract data from host processes only on-demand / when at least one
 	// of the specified descriptors requires host data
 	for _, d := range aixDescriptors {
 		if d.onHost {
-			setHostProcesses(pid)
+			ctx.hostProcesses, _ = hostProcesses(pid)
```

Comment:
You don't care if this fails?

Reply:
It has been ignored prior to this change. Now, we return the errors.

Reply:
... I mean: I updated the code and now we return the errors.